### PR TITLE
Empty testcase bug

### PR DESCRIPTION
**Pull request by pepop99**
See #9

### CODE REVIEW

**Review by Sanidhya0077 — DISMISSED** [review]

File: src/test/java/com/clevertap/maven/plugins/supertest/SuperTestMavenPluginTest.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+import java.util.List;
+import java.util.Map;
+
+import static junit.framework.Assert.assertEquals;
+
+class SuperTestMavenPluginTest {
+    @Test
+    void createRerunCommandTest() throws IOException, ParserConfigurationException, SAXException  {
+        SuperTestMavenPlugin bv = new SuperTestMavenPlugin();
+        final String classXML1 = "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n"
```
Comment (judepereira) [inline src/test/java/com/clevertap/maven/plugins/supertest/SuperTestMavenPluginTest.java]: No long strings. Please create files in the resources directory, and load them here instead.

File: src/main/java/com/clevertap/maven/plugins/supertest/SuperTestMavenPlugin.java [hunk 2]
```diff
@@ -150,7 +150,12 @@ public String createRerunCommand(Map<String, List<String>> classnameToTestcaseLi
         for (String className : classnameToTestcaseList.keySet()) {
             List<String> failedTestCaseList = classnameToTestcaseList.get(className);
             if (!failedTestCaseList.isEmpty()) {
-                retryRun.append(className).append("#");
+                retryRun.append(className);
+                if(failedTestCaseList.contains("")) {
```
Comment (judepereira) [inline src/main/java/com/clevertap/maven/plugins/supertest/SuperTestMavenPlugin.java:154]: What if it it contains `"" and "testName"`? Should the condition be updated? To ignore empty entries in the list?

**Review by judepereira — CHANGES_REQUESTED** [review]

File: src/main/java/com/clevertap/maven/plugins/supertest/SuperTestMavenPlugin.java (hunk 2 above)
Comment (pepop99) [reply src/main/java/com/clevertap/maven/plugins/supertest/SuperTestMavenPlugin.java:154]: if it contains "" and "testName", we rerun the entire class. We don't need to specify the tests to run here. We rerun the entire class.

File: src/test/java/com/clevertap/maven/plugins/supertest/SuperTestMavenPluginTest.java (hunk 1 above)
Comment (pepop99) [reply src/test/java/com/clevertap/maven/plugins/supertest/SuperTestMavenPluginTest.java]: got it. Will make the change.

File: src/main/java/com/clevertap/maven/plugins/supertest/SuperTestMavenPlugin.java (hunk 2 above)
Comment (pepop99) [reply src/main/java/com/clevertap/maven/plugins/supertest/SuperTestMavenPlugin.java:154]: src/test/resources/FooTest.xml actually demonstrates this behaviour.

**Review by judepereira — APPROVED** [review]

**Pull request merged**